backend/slackbot/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Please set up ESLint for the Slackbot package and configure Travis CI to lint this module - see the reviews module for an example.